packages/fiori/src/SideNavigationSelectableItemBase.ts

```diff
@@ -83,6 +87,50 @@ class SideNavigationSelectableItemBase extends SideNavigationItemBase {
 	@property()
 	target?: string;
 
+	/**
+	 * Item design.
+	 *
+	 * **Note:** Items with "Action" design must not have sub-items.
+	 *
+	 * @public
+	 * @default "Default"
+	 * @since 2.7.0
+	 */
+	@property()
+	design: `${SideNavigationItemDesign}` = "Default";
+
+	/**
```

Comment:
@Stoev please re-check this part of the documentation as it was significantly changed.